Flat/src/ModulesFatJets.cc

```diff
@@ -356,17 +349,27 @@ void PandaAnalyzer::FatjetBasics()
         } else {
           double smear=1, smearUp=1, smearDown=1;
           ak8JERReader->getStochasticSmear(pt,eta,event.rho,smear,smearUp,smearDown);
-
           gt->fj1PtSmeared = smear*gt->fj1Pt;
           gt->fj1PtSmearedUp = smearUp*gt->fj1Pt;
           gt->fj1PtSmearedDown = smearDown*gt->fj1Pt;
-
           gt->fj1MSDSmeared = smear*gt->fj1MSD;
+          if (analysis->hbb) { // "In the Higgs group, we do measurements." ~A. Rizzi
```

Comment:
I get the joke, but I'd rather not have someone stumble upon a snarky comment about them in my repository.